hack/jenkins/e2e.sh

```diff
@@ -940,22 +940,22 @@ case ${JOB_NAME} in
           )"}
     ;;
 
-  # kubernetes-upgrade-gke-stable-latest
+  # kubernetes-upgrade-gke-stable-current-release
   #
   # This suite:
   #
   # 1. launches a cluster at release/stable,
-  # 2. upgrades the master to release/latest,
+  # 2. upgrades the master to CURRENT_RELEASE_PUBLISHED_VERSION,
```

Comment:
Could you add a comment on  CURRENT_RELEASE_PUBLISHED_VERSION that it refers to the latest ci build on the current release branch? I thought it was stable based on the "PUBLISHED". Might be worth renaming it to CURRENT_RELEASE_CI_VERSION.